components/email/src/Email.svelte

```diff
@@ -448,10 +449,24 @@
       if (trashLabelID) {
         const existingLabelIds = activeThread.labels?.map((i) => i.id) || [];
         activeThread.label_ids = [...existingLabelIds, trashLabelID];
+        await saveActiveThread();
       } else if (trashFolderID) {
         activeThread.folder_id = trashFolderID;
```

Comment:
Do we still keep this line?

Reply:
@yifanplanet yes, if the account uses labels, we want to move thread instead. As this workaround is only for non-gmail accounts which use folders.